include/AutomationEditor.h

```diff
@@ -55,7 +55,8 @@ class AutomationEditor : public QWidget, public JournallingObject
 	Q_PROPERTY(QColor barLineColor READ barLineColor WRITE setBarLineColor)
 	Q_PROPERTY(QColor beatLineColor READ beatLineColor WRITE setBeatLineColor)
 	Q_PROPERTY(QColor lineColor READ lineColor WRITE setLineColor)
-	Q_PROPERTY(QColor vertexColor READ vertexColor WRITE setVertexColor)
+	Q_PROPERTY(QColor nodeInValueColor READ nodeInValueColor WRITE setNodeInValueColor)
+	Q_PROPERTY(QColor nodeOutValueColor READ nodeOutValueColor WRITE setNodeOutValueColor)
```

Comment:
Since the READ and WRITE methods are just `return x` and `x = y`, Qt allows the use of the `MEMBER class_var` that sets these default functions for the property.

We could change this (and others in this class and other classes) to:
```suggestion
	Q_PROPERTY(QColor nodeInValueColor MEMBER m_nodeInValueColor)
	Q_PROPERTY(QColor nodeOutValueColor MEMBER m_nodeOutValueColor)
```

Reply:
I added a mutex lock on each read/write method for those properties, so wouldn't I need to override the default Qt methods anyways?

Reply:
The node value shouldn't be changed after the initial load though, should it? I would assume the setters are called once each on startup, so no risk of concurrent access there. Getting from several places should be safe too since the value doesn't change.

Reply:
> The node value shouldn't be changed after the initial load though, should it?

So the initial load does some weird things, and during my tests on PianoRoll they were called like 3 or 4 times. That being said, I don't believe they need the mutex locks.

Reply:
If they are called more then once isn't there a possibility of concurrent access? Isn't it a good idea to keep the mutexes just in case? Would also make the path clear if we decide to allow customizing those properties/colors in real time

Reply:
READ is called a lot, WRITE is called once iirc.